ddl/src/main/java/com/adaptivescale/rosetta/ddl/targets/kinetica/KineticaDDLGenerator.java

```diff
@@ -1,10 +1,7 @@
 package com.adaptivescale.rosetta.ddl.targets.kinetica;
 
 import com.adaptivescale.rosetta.common.annotations.RosettaModule;
-import com.adaptivescale.rosetta.common.models.Column;
-import com.adaptivescale.rosetta.common.models.Database;
-import com.adaptivescale.rosetta.common.models.ForeignKey;
-import com.adaptivescale.rosetta.common.models.Table;
+import com.adaptivescale.rosetta.common.models.*;
```

Comment:
Can we make sure we don't use * import for the `common.models`.

Reply:
Yes this is resolved on both scenarios, thanks for your recommendation